README.md

```diff
@@ -1,14 +1,58 @@
-# Prometheus Integration Pack
+# prometheus integration pack v0.2.2
+
+> Interact with Prometheus HTTP API
+Daniel Chamot <daniel@nullkarma.com>
+
+### Contributors
+- Carlos <nzlosh@yahoo.com>
+
 
 ## Configuration
 
-```yaml
-url: http://prometheus.tld:9090
-```
+The following options are required to be configured for the pack to work correctly.
 
+| Option | Type | Required | Secret | Description |
+|---|---|---|---|---|
+| `url` | string | True |  | Prometheus Url |
 ## Actions
 
-* `alert.aggregation`
-* `query` - query Prometheus API
-* `series` - query Prometheus Series API
-* `Targets` - query Prometheus Targets API
+
+The pack provides the following actions:
+
+### targets
+_Prometheus Targets API_
+
+| Parameter | Type | Required | Secret | Description |
+|---|---|---|---|---|
+| `url` | string | False | default | _Prometheus URL_ |
+### query
+_Instant Query Prometheus API_
+
+| Parameter | Type | Required | Secret | Description |
+|---|---|---|---|---|
+| `query` | string | default | default | _Unavailable_ |
+| `url` | string | default | default | _Prometheus Url_ |
+### series
+_Query Prometheus Series API_
+
+| Parameter | Type | Required | Secret | Description |
+|---|---|---|---|---|
+| `queries` | object | default | default | _Unavailable_ |
+| `url` | string | False | default | _Prometheus URL_ |
+### alert.aggregation
+__
+
+| Parameter | Type | Required | Secret | Description |
+|---|---|---|---|---|
+| `alerts` | array | default | default | _Unavailable_ |
+| `parent` | string | default | default | _Label group parent / outer label_ |
+| `child` | string | default | default | _Label group child_ |
+
+
+
+## Sensors
+
+There are no sensors available for this pack.
+
+
+<sub>Documentation generated using [pack2md](https://github.com/nzlosh/pack2md)</sub>
```

Comment:
I'm fine with the link & credits about pack2md, which is a helpful tool in this particular situation.

However what if someone changes the Readme without using this tool and vice-versa changing the pack meta without updating Readme? Having README.md such detailed might lead us to such cases.

Reply:
That's a good point about editing the README.md file.  If we can't count of developers using the tool consistently, perhaps we could look at automating the generation of README.md on merge master?